packages/models/package.json

```diff
@@ -2,6 +2,13 @@
   "name": "@code-pushup/models",
   "version": "0.49.0",
   "license": "MIT",
+  "description": "Model definitions and validators for the [Code PushUp CLI](https://github.com/code-pushup/cli/tree/main/packages/cli/README.md)",
```

Comment:
I don't think NPM supports Markdown formatting in `description` fields.

Reply:
removed